CHANGELOG.md

```diff
@@ -1,5 +1,8 @@
 # Change Log - Power BI Custom Visuals API
 
```

Comment:
please add a log for 5.1.0 (maybe was missed in merge)

Reply:
Done. And 4.7.0 as well.